examples/oneapi/cpp/source/kmeans/kmeans_lloyd_dense_batch.cpp

```diff
@@ -17,12 +17,33 @@
 #include "example_util/utils.hpp"
 #include "oneapi/dal/algo/kmeans.hpp"
 #include "oneapi/dal/io/csv.hpp"
+#include "oneapi/dal/network/mpi.hpp"
 
 using namespace oneapi;
 
+#define MPI
+
+int main(int argc, char* argv[]) {
+    int myRank = 0;
+    #ifdef MPI
+        MPI_Init(&argc, &argv);
```

Comment:
No way for user to select a particular communicator/network? Direct call to MPI to obtain ranks? Something like:
MPI_network network;
int myRank = network.get_communicator()->rank();
env->setDefaultNetwork(network);
algo.compute();